spec/dummy/db/migrate/20220623194739_add_initial_data.rb

```diff
@@ -1,13 +0,0 @@
-# frozen_string_literal: true
-
-class AddInitialData < ActiveRecord::Migration[6.1]
```

Comment:
should this be in a seeds flle?

Reply:
Sure but I'll not use this fake data anymore then I deleted